htdocs/wp-config.php

```diff
@@ -68,11 +68,27 @@
 define('FORCE_SSL_ADMIN', true);
 
 /**
- * Custom Settings
+ * Disable automatic updates and use Seravo updates instead
  */
-define('AUTOMATIC_UPDATER_DISABLED', true); /* automatic updates are handled by wordpress-palvelu */
-define('DISALLOW_FILE_EDIT', true); /* disable the theme/plugin file editor */
-define('PLL_COOKIE', false); /* allow caching sites with polylang, disable if weird issues occur */
+define('AUTOMATIC_UPDATER_DISABLED', true);
+
+/*
+ * Disable the theme/plugin file editor for security reasons
+ */
+define('DISALLOW_FILE_EDIT', true);
+
+/*
+ * Prevent Polylang from setting extra language cookies, so that the HTTP cache
+ * is not busted in vain. The language is anyway in the request URL (e.g. /en/)
+ * so extra language cookies are not needed.
+ */
+define('PLL_COOKIE', false);
+
+/**
+ * Standardize cache location to have as much as possible in wp-content/cache
+ * so that it is easier to develop tools to accelerate or purge caches.
+ */
+define('WPML_CACHE_PATH_ROOT', dirname(__DIR__) . '/htdocs/wp-content/cache/wpml/');
```

Comment:
Does this work OK with sites that have some custom directory structures, and where files are actually located outside `/data/wordpress/htdocs` and then some folders have been symlinked?